### ISSUE TITLE
Whenever incorrectly parses the crontab expression.

### ISSUE THREAD

**Issue by io-german** [javan/whenever#704]
Given this example of code we expect that command will be scheduled to run each Sunday on noon.

```ruby
# every Sunday at 00:00
every '0 12 * * SUN' do
  runner "some funny command"
end
```

But ```whenever``` command shows that this expression is parsed incorrectly. The time of execution is set to midnight:

```bash
$ whenever
0 0 * * 0 /bin/bash -l -c 'cd /Users/user/projects/project && bundle exec bin/rails runner -e production '\''some funny command'\''
```

After playing a while with expression I have found that it doesn't matter what hour of day I set - it's just ignored. However if I change stringified day of week to a number the problem solves.

Just like that:

```ruby
# every Sunday at 00:00
every '0 12 * * 0' do
  runner "some funny command"
end
```

```bash
$ whenever
0 12 * * 0 /bin/bash -l -c 'cd /Users/user/projects/project && bundle exec bin/rails runner -e production '\''some funny command'\''
```

**Comment by crivotz** [comment]
Same here, he set always 0 0 * * 0

```
every :day, :at => '00am' do
rake 'do_stuff'
end

every :day, :at => '14am' do
rake 'blazer:send_failing_check'
end
```

**Comment by benlangfeld** [comment]
Should be fixed by #711. Please try again with whenever v0.10.0.

**Comment by crivotz** [comment]
Thanks
